b4f0c3: Adjust meta viewport applicability based on properties defined (#1835)
act-rules/act-rules.github.io: _rules/meta-viewport-b4f0c3.md

## Applicability

> This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport`. In addition, the `content` [attribute value][] has the `user-scalable` or `maximum-scale` properties.

Reviewer: This is better aligned with other rules and eliminates any eventual ambiguity when the attribute includes the two properties (in case the "or" is not understood as a "logical or").

```suggestion
This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
- the `content` [attribute value][] has the `user-scalable` property; or
- the `content` [attribute value][] has the `maximum-scale` property.
```

## Applicability

> This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
> - the `content` [attribute value][] has the `user-scalable` property; or

Reviewer 2: ```suggestion
This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:

- the `content` [attribute value][] has the `user-scalable` property; or
```

## Applicability

> This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport`. In addition, when the `content` attribute value is mapped to a list of property/value pairs in a user-agent specific manner, one of the properties is either `user-scalable` or `maximum-scale`.

Reviewer 3: I'm quite reluctant in using "in a user-agent specific manner" since it sounds pretty ambiguous if user agents don't all do it the same way…
CSS is listing [this algorithm](https://drafts.csswg.org/css-device-adapt/#parsing-algorithm) for parsing it, which seems to be the iOS/Safari one. But also sounds pretty OK given what's normally in that list. At least, saying that the attribute is parsed that way would remove the ambiguity.

#### Passed Example 2

> This viewport `meta` element allows users to scale content up to 600% because it has `maximum-scale` set to 6.0.

Reviewer 2: ```suggestion
This viewport `meta` element allows users to scale content up to 200% because it has `maximum-scale` set to 6.0.
```

#### Passed Example 2

> This viewport `meta` element allows users to scale content up to 200% because it has `maximum-scale` set to 6.0.

Reviewer 2: ```suggestion
This viewport `meta` element allows users to scale content up to 200% because it has `maximum-scale` set to 2.0.
```

</html>
```

#### Inapplicable Example 3

> This viewport `meta` element does not prevent user scaling because it does not specify the `maximum-scale` nor `user-scalable` values.

Reviewer: As it stands, it sounds like a Passed Example. 

```suggestion
This viewport `meta` element does not specify the `maximum-scale` nor `user-scalable` values.
```

</html>
```

#### Inapplicable Example 4

> This viewport `meta` element does not prevent user scaling because it does not specify the `maximum-scale` nor `user-scalable` values.

Reviewer: What's the difference for Inapplicable Example 3?

Author: I think @WilcoFiers asked me to include these changes? I honestly don't remember changing the examples, but rather just moving one of them from passed to N/A.

Reviewer 2: Talked to Carlos. The point is that both example 3 and 4 have a content attribute, but he said he doesn't mind it. The two examples are different enough. We're good to move forward with this.
